packages/base/src/utils.ts

```diff
@@ -45,3 +45,36 @@ export function addHasRemoveClass(element: HTMLElement) {
     hasClass: (className: string) => element.classList.contains(className),
   };
 }
+
+/**
+ * Event listeners suport `passive` option
+ */
+export let supportsPassiveEventListener = false;
+/**
+ * Event listeners support `once` option
+ */
+export let supportsOnceEventListener = false;
```

Comment:
Doesn't look like we're using the `once` or `capture` probes, remove them for now? Or do you expect we'll need them for something else sometime soon?

Reply:
I can probably remove them